### ISSUE TITLE
Chore: Add better response for requests with missing Authentification header

### ISSUE THREAD

**Issue by saerdnaer** [swrlab/ard-eventhub#682]
During refactoring of our client code, I came across following behaviour where the response is error 400 instead of 401 when the request is missing an `Authentication` header:

```
curl -v -X POST https://eventhub-ingest-test.ard.de/events/de.ard.eventhub.v1.radio.track.playing
…
> POST /events/de.ard.eventhub.v1.radio.track.playing HTTP/2
> Host: eventhub-ingest-test.ard.de
> User-Agent: curl/8.4.0
> Accept: */*
> 
< HTTP/2 400 
< date: Mon, 27 May 2024 12:53:09 GMT
< content-type: application/json; charset=utf-8
< content-length: 14
< x-service: ard-eventhub-ingest/1.7.3

{"trace":null}
```

According to https://eventhub-ingest.ard.de/openapi/#/events/eventPostV1RadioTrackPlaying I would expect a 401 and not an nearly empty object `{"trace":null}`:
![image](https://github.com/swrlab/ard-eventhub/assets/40266/73b1d50c-93e0-4440-a6a1-199173207266)

**Comment by frytg** [comment]
Will be fixed in `v1.8.0` in https://github.com/swrlab/ard-eventhub/commit/ce18dbdc05eaa444e5cf75fc3b351957b8a7ae7a

GET will still result in 400, since `GET method not allowed` takes priority over the missing header in that case. POST will be 401.

**Comment by frytg** [comment]
@saerdnaer now in production as well
